packages/flutter/test/widgets/autocomplete_test.dart

```diff
@@ -421,6 +421,130 @@ void main() {
     expect(textEditingController.text, lastOptions.elementAt(0));
   });
 
+  group('optionsViewOpenDirection', () {
+    testWidgets('unset (default behavior): open downward', (WidgetTester tester) async {
+      await tester.pumpWidget(
+        MaterialApp(
+          home: Scaffold(
+            body: Center(
+              child: RawAutocomplete<String>(
+                optionsBuilder: (_) => <String>['a'],
+                fieldViewBuilder: (_, TextEditingController controller, FocusNode focusNode, __) {
+                  return TextField(controller: controller, focusNode: focusNode);
+                },
+                optionsViewBuilder: (_, __, ___) => const Placeholder())))));
+      await tester.showKeyboard(find.byType(TextField));
+      final Offset fieldBottomLeft = tester.getBottomLeft(find.byType(TextField));
+      final Offset optionsViewTopLeft = tester.getTopLeft(find.byType(Placeholder));
+      final Offset optionsViewCenter = tester.getCenter(find.byType(Placeholder));
+      expect(fieldBottomLeft, offsetMoreOrLessEquals(optionsViewTopLeft));
+      expect(optionsViewCenter.dy, greaterThan(optionsViewTopLeft.dy));
```

Comment:
This last check boils down to just saying that the options view doesn't have zero height — the center of any box will be below the top left of the same box by half the box's height.

I think it's OK to leave that check out. Particularly if the options view is something like a Text widget where it'd clearly be weird if it somehow got zero height.